docs/using-mycroft-ai/basic-commands.md

```diff
@@ -1,104 +1,328 @@
 ---
 description: >-
-  Now that your Mycroft is now **Paired** with home.mycroft.ai, you can begin
-  issuing commands. Try these to get started.
+  Now that your Mycroft Device is Paired with home.mycroft.ai, you may begin
+  issuing commands. Try these to get started.  
 ---
-
 # Basic Commands
 
-## Default **Skills** installed with Mycroft
-
-### Alarms
-
-> Hey Mycroft, set an alarm for two hours
->
-> Hey Mycroft, set an alarm for 3pm
-
-### Audio record
-
-> Hey Mycroft, record
-
-### Configuration
-
-> Hey Mycroft, configuration update
-
-### DuckDuckGo
-
-> Hey Mycroft, what is Frankenstein?
->
+You issue a command to Mycroft by Speaking the **Wake Word** (e.g. "Hey Mycroft"), awaiting aknowledgement from Mycroft, and then Speaking the command itself (aka the **Utterance**). 
+
+When you issue a command, you *intend* for Mycroft to react in a certain way.  Your **Utterance** contains an **Intent**.  Mycroft uses natural language processing to compare the words in your **Utterance** to expected phrases or keywords as supplied by the installed **Skills**.  When Mycroft determines that there is a *match* between your **Utterance** and a specific expectation (the phrases/keywords), the **Skill** that supplied the matching expectation is notified so that it can respond to your command.
+
+Mycroft comes with a collection of **Skills** installed by default.  Each **Skill** supplies a set of **Intents**, and each **Intent** has a set of expected phrases or keywords.  Each of the default **Skills** is introduced below with a brief description, followed by a sampling of expected **Utternances** (usually grouped by **Intent**). Additional **Skills** may be added to your Mycroft Device; please see [Installing New Skills](https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/installing-new-skills).  
+
+Have fun exploring natural language processing by trying these commands with variations to the wording and phrasing. 
+###
+## The Default **Skills**
+### Alarm - *Manages a collection of alarms.  Weekly repetition is supported.*
```

Comment:
As the descriptions aren't really part of the heading, they should probably go on a new line. This will also keep the automatically generated table of contents cleaner and more readable.

Reply:
Will do.  Good point.